src/physical/management/execution_plan.rs

```diff
@@ -815,6 +888,16 @@ impl ExecutionTree {
                 assert!(permutation.is_identity());
                 Self::satisfy_leapfrog_recurisve(subnode.clone(), permutation.clone());
             }
+            ExecutionOperation::Subtract(subnode_main, subnodes_subtract, _) => {
+                // TODO: A few other changes are needed to make this a bit simpler.
+                // Will update it then
```

Comment:
What are the changes?

Reply:
What is needed here is a way to change this operation node such that it outputs columns in a different order according to the permutation. 

I'm too lazy now to think this through  for this operation (and the others that have a similar comment). Also, I most likely will rework how this works entirely anyways, so it might end up not mattering in the end.